modules/efk-logging-kibana-tolerations.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-kibana.adoc
+
+[id="efk-logging-kibana-tolerations_{context}"]
+= Using tolerations to control the Kibana pod placement
+
+You can ensure which nodes the Kibana pod runs on
```

Comment:
it looks not very clear. Something like "You can control on which node the kibana pod runs using tolerations on the pod" sounds clear